### PR TITLE
Tool suggestion and addition for breaking JWT 

### PR DESCRIPTION
**Pull request by rishuranjanofficial**
This PR covers issue #<issue number>.

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Tool suggestion and addition for breaking JWT 
- changing the algorithm of the input JWT to 'None' and 'HS256/384/512' automatically.

Thank you for your contribution!

### CODE REVIEW

File: Testing_for_APIs.md [hunk 1]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
 
 Step 8: Resend one of the operations from step 6 with an interception proxy (Burp Repeater, for example). This will send to the server a request with the session ID or token that was invalidated in step 7.
 
+#### For performing the step 2 and 5, use the following python tool which can perform the task to break JWT tokens faster
```
Comment (kingthorin) [inline Testing_for_APIs.md:124]: This whole be a proper heading. (Or not be a heading at all).

File: Testing_for_APIs.md [hunk 2]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
...
+
+Step 1: Clone the repository [JWTweak](https://github.com/rishuranjanofficial/JWTweak)
+
+Step 2: Navigate to cloned repository 
+
+Step 3: Run `python3 JWTweak.py`
```
Comment (kingthorin) [inline Testing_for_APIs.md:130]: The extra spaces are unnecessary.

File: Testing_for_APIs.md [hunk 3]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
...
+
+Usage - Detects the algorithm of input JWT Token and provide options to generate the new JWT token based on the user selected algorithm.
```
Comment (kingthorin) [inline Testing_for_APIs.md:132]:
```suggestion
Detects the algorithm of input JWT Token and provide options to generate the new JWT token based on the user selected algorithm.
```

Usage was shown in step 3 above.

File: Testing_for_APIs.md [hunk 4]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
 
 Step 8: Resend one of the operations from step 6 with an interception proxy (Burp Repeater, for example). This will send to the server a request with the session ID or token that was invalidated in step 7.
 
+#### For performing the step 2 and 5, use the following python tool which can perform the task to break JWT tokens faster
+
+Step 1: Clone the repository [JWTweak](https://github.com/rishuranjanofficial/JWTweak)
+
+Step 2: Navigate to cloned repository
```
Comment (kingthorin) [inline Testing_for_APIs.md:128]:
```suggestion
Step 2: Navigate to cloned repository
```

File: Testing_for_APIs.md (hunk 1 above)
Comment (rishuranjanofficial) [reply Testing_for_APIs.md:124]: decided not to be heading

File: Testing_for_APIs.md (hunk 2 above)
Comment (rishuranjanofficial) [reply Testing_for_APIs.md:130]: noted

File: Testing_for_APIs.md (hunk 3 above)
Comment (rishuranjanofficial) [reply Testing_for_APIs.md:132]: changed

**Comment by rishuranjanofficial** [comment]
all suggestions are done. Please review

File: Testing_for_APIs.md [hunk 5]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
 
 Step 8: Resend one of the operations from step 6 with an interception proxy (Burp Repeater, for example). This will send to the server a request with the session ID or token that was invalidated in step 7.
 
+For performing the step 2 and 5, use the following python tool which can perform the task to break JWT tokens faster
+
+Step 1: Clone the repository [JWTweak](https://github.com/rishuranjanofficial/JWTweak)
+
```
Comment (kingthorin) [inline Testing_for_APIs.md:127]:
```suggestion
```

File: Testing_for_APIs.md [hunk 6]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
...
+Step 2: Navigate to cloned repository
+
```
Comment (kingthorin) [inline Testing_for_APIs.md:129]:
```suggestion
```

File: Testing_for_APIs.md [hunk 7]
```diff
@@ -121,6 +121,16 @@ Step 7: Log out or change password of concurrent working user.
...
+Step 3: Run `python3 JWTweak.py`
+
+JWTweak can also detects the algorithm of input JWT Token and provide options to generate the new JWT token based on the user selected algorithm.
```
Comment (kingthorin) [inline Testing_for_APIs.md:132]:
```suggestion
JWTweak can also detect the algorithm of input JWT Token and provide options to generate the new JWT token based on the user selected algorithm.
```

File: Testing_for_APIs.md [hunk 8]
```diff
@@ -121,6 +121,15 @@ Step 7: Log out or change password of concurrent working user.
 
 Step 8: Resend one of the operations from step 6 with an interception proxy (Burp Repeater, for example). This will send to the server a request with the session ID or token that was invalidated in step 7.
 
+For performing the step 2 and 5, use the following python tool which can perform the task to break JWT tokens faster
+
+Step 1: Clone the repository [JWTweak](https://github.com/rishuranjanofficial/JWTweak)
+
```
Comment (kingthorin) [inline Testing_for_APIs.md:127]:
```suggestion
```

**Comment by rishuranjanofficial** [comment]
first review is done. 2nd review is pending.

**Comment by kingthorin** [comment]
Sorry this doesn't really fly. The tool has only existed for 17 days, has zero releases, and 1 or 2 contributors.

We're not trying to mention or advertise every tool under the sun, things that get mentioned or pointed out should be well established within the industry.

**Comment by rishuranjanofficial** [comment]
Hi kingthorin,

I totally agree with you and I am not suggesting or adding this tool for advertising. I thought this tool can really help our people in JWT related testcases which is becoming prominent these days. I am closing this PR then. 

Anyway I am loving working and will continuing working for wstg 😊 

Thank you!

**Pull request closed without merging**